experimental/yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -51,6 +51,8 @@ class FlowableOperator : public Flowable<D> {
       subscriber_->onSubscribe(Reference<::yarpl::flowable::Subscription>(this));
     }
 
+    void onNext(U) override {}
```

Comment:
If you think `Subscriber` is too abstract to have a default implementation of `onNext()` then I think `FlowableOperator` shouldn't have one either.